templates/set_bonus_tokens_to_zero.cdc

```diff
@@ -0,0 +1,14 @@
+
+/// Set the bonus tokens to zero.
+/// The reward calculation should be based on the total without subtracting any bonus tokens
+/// of bonus tokens that have yet to be burned
+transaction() {
+
+    prepare(signer: AuthAccount) {
+        let existingBonusAmount = signer.load<UFix64>(from: /storage/FlowBonusTokenAmount)
```

Comment:
@bjartek is this even needed in that case?

Reply:
and thank you for the previous comments.

Reply:
```If there is already an object stored under the given path, the program aborts.``` yes you need to load it first.